libvast/src/system/exporter.cpp

```diff
@@ -63,8 +63,20 @@ void ship_results(exporter_actor::stateful_pointer<exporter_state> self) {
     st.query_status.cached -= rows;
     st.query_status.requested -= rows;
     st.query_status.shipped += rows;
-    auto transformed = self->state.transformer.apply(std::move(slice));
-    self->anon_send(st.sink, std::move(*transformed));
+    auto add_failed = self->state.transformer.add(std::move(slice));
+    if (add_failed) {
```

Comment:
```suggestion
    if (auto err = self->state.transformer.add(std::move(slice))) {
```

This is a common design pattern in our code base.